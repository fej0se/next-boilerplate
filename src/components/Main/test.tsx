import { render, screen } from '@testing-library/react'

import Test from '.'

describe('<Test />', () => {
  it('should render the heading', () => {
    const { container } = render(<Test />)

    expect(
      screen.getByRole('heading', { name: /This is a component test/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
