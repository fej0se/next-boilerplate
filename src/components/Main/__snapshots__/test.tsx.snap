// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Test /> should render the heading 1`] = `
<main>
  <h1>
    This is a component test
  </h1>
</main>
`
